src/coreclr/tools/ILVerify/README.md

```diff
@@ -48,24 +48,39 @@ The test project itself is under [src/tests/ilverify](../../../tests/ilverify)
 
 General instructions to build this library can be found [here](https://github.com/dotnet/runtime/blob/main/docs/workflow/testing/coreclr/testing.md).
 
+As quick snippet which should be enough to build CoreCLR
+```
+./build.cmd -s clr.native+clr.corelib+clr.tools+clr.nativecorelib+libs -c release
+```
+
 As the test project is marked with priority=1, simply building the test projects from the root of the project is not enough. For the initial build of priority=1 in release mode, run the following:
 
 ```sh
 src/tests/build.(cmd/sh) release -priority=1
 ```
 
+or significantly faster use only
```

Comment:
The existing instructions can be deleted. I guess they were written before the test build had `tree` option.